lib/bolt_ext/server.rb

```diff
@@ -48,6 +51,22 @@ class TransportAPI < Sinatra::Base
     content_type :json
 
     body = JSON.parse(request.body.read)
+    # Some windows ruby interpreters will not preserve integer type when parsing JSON
```

Comment:
We should figure out what's actually going on here...

Reply:
The environment variable for BOLT_WINRM_PORT port was a string in https://github.com/puppetlabs/bolt/blob/0c211fd33b8e0e9defe01c2df6e30e00eb187def/spec/lib/bolt_spec/conn.rb#L27
(facepalm)